codewof/general/management/commands/sampledata.py

```diff
@@ -77,3 +95,117 @@ def handle(self, *args, **options):
         StudyFactory.create_batch(size=5)
         StudyGroupFactory.create_batch(size=15)
         print('Research studies loaded.')
+
+        Badge.objects.create(
+            id_name='create-account',
+            display_name='Created an account!',
+            description='Created your very own account',
+            icon_name='img/icons/badges/icons8-badge-create-account-48.png',
+            badge_tier=0
+        )
+
+        Badge.objects.create(
+            id_name='questions-solved-1',
+            display_name='Solved one question!',
+            description='Solved your very first question',
+            icon_name='img/icons/badges/icons8-question-solved-black-50.png',
+            badge_tier=1
+        )
+
+        Badge.objects.create(
+            id_name='questions-solved-5',
+            display_name='Solved five questions!',
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/65398607-68c18e80-de0c-11e9-9668-c7a90c0b427c.png)

Would be cool if the icon could be displayed with this message